01-wasmcloud_basics/wash_intro/finish.md

```diff
@@ -0,0 +1,3 @@
+## Congratulations
+
+Thanks for trying out the wasmCloud runtime. We are a tight knit community that would love you have you join us. Please checkout our website [wasmcloud.com](wasmcloud.com) and join us on Slack!
```

Comment:
I never see this in the tutorial, does it need to be linked somewhere?

Reply:
Accidentally disabled it....will fix.